relay-dynamic-config/src/global.rs

```diff
@@ -38,6 +38,16 @@ pub struct GlobalConfig {
         skip_serializing_if = "is_default"
     )]
     pub options: Options,
+
+    /// Configuration for global metrics extraction rules.
+    ///
+    /// These are merged with rules in project configs before
+    /// applying.
+    #[serde(
+        deserialize_with = "default_on_error",
```

Comment:
This will log an error when serialization fails.

Reply:
Mh maybe we should do an explicit error boundary and skip extraction in an error case, like we do for inbound filters. This should prevent and futureproof all possible managed/self hosted Relay issues.